src/Chainweb/Version.hs

```diff
@@ -984,6 +982,23 @@ chainweb215Pact aoa v h = case aoa of
     go f (FastTimedCPM g) | g == petersonChainGraph = f 35
     go f _ = f 10
 
+-- | Pact and coin contract changes for Chainweb 2.15
+--
+chainweb216Pact
+    :: AtOrAfter
+    -> ChainwebVersion
+    -> BlockHeight
+    -> Bool
+chainweb216Pact aoa v h = case aoa of
+    At -> go (==) v h
+    After -> go (<) v h
+  where
+    go f Mainnet01 = f 2988358 -- 2022-09-02 00:00:00+00:00
```

Comment:
do these need to be checked/verified?

Reply:
Yes they do.

Reply:
I think, there are check boxes for it on the release candidate PR and the release checklist. If not can you add it?